src/pages/CustomerInvoicePage.js

```diff
@@ -200,9 +200,9 @@ export const CustomerInvoice = ({
 };
 
 const mapDispatchToProps = (dispatch, ownProps) => {
-  const { transaction } = ownProps || {};
-  const { otherParty } = transaction;
-  const hasMasterLists = otherParty.masterLists.length > 0;
+  const { transaction } = ownProps?.navigation?.route?.param || {};
+  const { otherParty } = transaction || {};
```

Comment:
`??`.

![lol](https://user-images.githubusercontent.com/43223637/75945355-debf3b00-5efe-11ea-9d48-7c2418c5a51a.gif)

Reply:
heh, everytime! will fix them up in the next PR